Sources/Rendering/Core/Camera/index.js

```diff
@@ -426,6 +426,38 @@ export function extend(publicAPI, model, initialValues = {}) {
 
   // Build VTK API
   macro.obj(publicAPI, model);
+
+  if (model.position === null) {
+    model.position = [0, 0, 1];
+  }
+  if (model.focalPoint === null) {
+    model.focalPoint = [0, 0, 0];
+  }
+  if (model.viewUp === null) {
+    model.viewUp = [0, 1, 0];
+  }
+  if (model.directionOfProjection === null) {
+    model.directionOfProjection = [0, 0, -1];
+  }
+  if (model.clippingRange === null) {
+    model.clippingRange = [0.01, 1000.01];
+  }
+  if (model.windowCenter) {
+    model.windowCenter = [0, 0];
+  }
+  if (model.viewPlaneNormal === null) {
+    model.viewPlaneNormal = [0, 0, 1];
+  }
+  if (model.screenBottomLeft === null) {
+    model.screenBottomLeft = [-0.5, -0.5, -0.5];
+  }
+  if (model.screenBottomRight === null) {
+    model.screenBottomRight = [0.5, -0.5, -0.5];
+  }
+  if (model.screenTopRight === null) {
```

Comment:
instead of checking for exactly null, I'm wondering if it would be better to check more loosely.

```
if (!model.screenTopRight) {
```